### ISSUE TITLE
Cannot use radio type checkbox as form field dependent

### ISSUE THREAD

**Issue by krisztianodor** [Semantic-Org/Semantic-UI#6245]
This part of the semantic.js should return true if a radio type dependency is unchecked, but only the checkbox type is handled:

```javascript
empty: function($field) {
  if(!$field || $field.length === 0) {
    return true;
  }
  else if($field.is('input[type="checkbox"]')) {
    return !$field.is(':checked');
  }
  else {
    return module.is.blank($field);
  }
},
```

**Comment by hammy2899** [comment]
Can you provide a test case by forking this [JSFiddle](https://jsfiddle.net/ca0rovs3/) to help reproduce your issue

**Comment by tmentink** [comment]
Here is the [JSFiddle](https://jsfiddle.net/mwhiterabbit/6exxn5s7/) from the old issue #4445 that has since been marked stale. 

I encountered this same issue and made a [pull request](https://github.com/Semantic-Org/Semantic-UI/pull/6191) about three weeks ago.

**Comment by lubber-de** [comment]
Implemented since Fomantic-UI 2.7.2 by fomantic/Fomantic-UI#372
